webapp/src/views/projects/translations/TranslationOpened.tsx

```diff
@@ -166,6 +179,24 @@ export const TranslationOpened: React.FC<Props> = ({
               onCancel={onCancel}
               onStateChange={onStateChange}
             />
+            <ShortcutsHint
+              items={[
+                {
+                  name: <T>{translationStates[nextState].translationKey}</T>,
```

Comment:
`TypeError: Cannot read properties of undefined (reading 'translationKey')` is thrown when I try to edit untranslated value. Please add test for this.